docs/aws/README.md

```diff
@@ -81,15 +112,20 @@ helm upgrade --install karpenter karpenter/karpenter \
   --namespace karpenter --set serviceAccount.create=false
 ```
 
-### (Optional) Enable Verbose Logging
-```bash
-kubectl patch deployment karpenter-controller \
-    -n karpenter --type='json' \
-    -p='[{"op": "replace", "path": "/spec/template/spec/containers/0/args", "value": ["--verbose"]}]'
-```
+## Provisioner
+
+A single Karpenter provisioner is capable of handling many different pod
+shapes. In other words, Karpenter eliminates the need to manage many different
+node groups. Karpenter makes scheduling and provisioning decisions based on pod
+attributes such as labels and affinity. 
+
+Create a simple default provisioner using the command below. This provisioner
+provides instances with the default certificate bundle, and the control plane
+endpoint url. 
+
+Importantly, the `ttlSecondsAfterEmpty` value configures Karpenter to
+deprovision empty nodes. 
 
-### Create a Provisioner
-Create a default Provisioner that launches nodes configured with cluster name, endpoint, and caBundle.
```

Comment:
We can now remove caBundle thanks to a recent PR!

Reply:
ok, I will remove `caBundle` def from the provisioner def